src/Symfony/Component/Routing/RouteCollection.php

```diff
@@ -89,6 +89,18 @@ public function all()
         return $this->routes;
     }
 
+    /**
+     * Check if a route exists.
+     *
+     * @param $name
+     *
+     * @return bool return true if the route exists, false if not
```

Comment:
should be replaced by a real return type
actually I think the docblock itself doesn't provide much info over the signature so it could be remvoed, isn't it?